requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy
+pandas
 quantities
 pyproj
 nose
```

Comment:
all the requirements should have an explicit version, you can patch this in a subsequent pull request.